.github/actions/report-job-status/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'Send VK Teams notification on failure'
+description: 'Send the message to VK Teams chat on failure'
+
+inputs:
+  job-steps:
```

Comment:
Could you please make this param the last one?

Reply:
done